build/azure-pipelines.yml

```diff
@@ -0,0 +1,61 @@
+﻿trigger:
+  branches:
+    include:
+    - master
+
+resources:
+   containers:
+     - container: windows
+       image: nventive/build-agent:vs16.4.0
+
+variables:
+- name: NUGET_VERSION
+  value: 5.4.0
+- name: VSTEST_PLATFORM_VERSION
+  value: 15.9.2
+- name: ArtifactName
+  value: Packages
+- name: SolutionFileName # Example: MyApplication.sln
+  value: LocationService.sln
+
+stages:
+- stage: Build
+  jobs:
+  - job: Windows
+    strategy:
+      maxParallel: 3
+      matrix:
+        Packages:
+          ApplicationConfiguration: Release
+          ApplicationPlatform: NuGet
+          GeneratePackageOnBuild: true
+
+    pool:
+      name: Container
```

Comment:
We can probably use 'Windows 1809' instead and change vs16.4.0 for vs16.6.0 on line 9.